tcf_website/static/reviews/review_form.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Cascading dropdown data population for the "new review" form
+ * in review.py and reviews/new.html
+ * Author: j-alicia-long, 11/22/2020
+*/
+
+// Executed when DOM is ready
+jQuery(function($) {
```

Comment:
Could you explain why you have `jQuery` here, instead of `$` as in everywhere else?

Reply:
It's just an equivalent jquery shorthand for `$(document).ready(function() { ... });`

I can change it to the other shorthand syntax `$(function() { ... });` if you prefer?

Reply:
Oh, I see. It's just that I've never seen it before. You can keep it.